Core/dune

```diff
@@ -1,16 +1,12 @@
 (library
  (name core_plugin)
  (public_name quickchick.core_plugin)
- (modules :standard)
- (libraries show_lib coq.plugins.ltac coq.plugins.extraction)
-)
+ (flags :standard -rectypes)
```

Comment:
This is sadly needed for all libraries linking to Coq; Dune could add this automatically however the feature request is not yet implemented.